cmd/pulumi-test-language/go.mod

```diff
@@ -88,7 +88,7 @@ require (
 	github.com/godbus/dbus/v5 v5.1.0 // indirect
 	github.com/gofrs/uuid v4.2.0+incompatible // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
-	github.com/golang-jwt/jwt/v5 v5.2.2 // indirect
+	github.com/golang-jwt/jwt/v5 v5.3.0 // indirect
```

Comment:
Are these changes actually necessary for the PR?  Or can the upgrade happen separately?